src/etc/inc/plugins.inc.d/dhcpd.inc

```diff
@@ -260,6 +260,13 @@ function dhcpd_radvd_configure($verbose = false, $blacklist = array())
             }
         }
 
+        if (!empty($dhcpv6ifconf['AdvLinkMTU'])) {
+            $overridemtu = $dhcpv6ifconf['AdvLinkMTU'];
+            if ($overridemtu < $mtu) {
+                $mtu = $overridemtu;
```

Comment:
Nice work here. radvd.conf states: "If specified, i.e. not 0, must not be smaller than 1280 and not greater than the maximum MTU allowed for this link (e.g. ethernet has a maximum MTU of 1500. See RFC 4864)."

One small issue though is that we should be logging the case when the overridemtu cannot be used (else condition).

Reply:
Added an else condition

https://github.com/opnsense/core/blob/898d34afefdf94f4093490651350483bb5f68d70/src/etc/inc/plugins.inc.d/dhcpd.inc#L267-L269